README.md

```diff
@@ -200,9 +213,8 @@ import (
 
 func initProjections(
     logger applogger.Logger,
-    rdbConn rdb.Conn,
-    config *bootstrap.Config,
-    customConfig *CustomConfig,
+	rdbConn rdb.Conn,
```

Comment:
there doesn't seem to be proper indentation here?

Reply:
thanks, just aligned all indentation